please.sh

```diff
@@ -2890,16 +2909,13 @@ bundle_pdbs () { # [--directory=<artifacts-directory] [--unpack=<directory>] [--
 		do
 			name=${package%-*}
 			version=$(echo "$versions" | sed -n "s/^$name //p")
+			dir2="$sdk_root/usr/src/MINGW-packages/$name"
 			case "$package" in
 			mingw-w64-*)
-				tar=mingw-w64-$arch-${package#mingw-w64-}-$version-any.pkg.tar.xz
-				dir2="$sdk64/usr/src/MINGW-packages/$name"
+				tar=mingw-w64-$pacman_arch-${package#mingw-w64-}-$version-any.pkg.tar.xz
 				;;
 			*)
-				tar=$package-$version-$arch.pkg.tar.xz
-				test i686 = $arch &&
-				dir2="$sdk32/usr/src/MSYS2-packages/$name" ||
-				dir2="$sdk64/usr/src/MSYS2-packages/$name"
```

Comment:
I fear that we still need to keep the `dir2` definition in these `case` arms, as the MSYS packages need to be found in `/usr/src/MSYS2-packages/<package-name>/`, not in ``/usr/src/MINGW-packages/<package-name>/`.

The original idea of this `dir2` business is that we cannot download the `-pdb` package for a `mingw-w64-git` package that we just built (and that we might not even deploy, if manual testing finds something incorrect in the Git artifacts and another workflow run has to be started).

So I think we still need this to look in the correct path, even if the common case  is that only a _MINGW_ package (concretely, `mingw-w64-git`) cannot be downloaded and needs to be found locally.

Reply:
Whoops, I completely overlooked the fact that it distinguished between `MINGW-packages` and `MSYS2-packages` in the first place. Apologies.

Given that `MSYS2-packages` can only be built for `x86_64` and `i686` at this point (as opposed to `MINGW-packages`, which supports `aarch64` as well), shall I just restore the original logic for this arm?

And for the `mingw-w64-*` arm, I think I can just replace `$sdk64/usr/src/MINGW-packages/$name` with `/usr/src/MINGW-packages/$name`, right? Same for the `$sdk64/git-cmd.exe` invocations below. If that works, we also don't have to introduce the `sdkarm64` variable above. WDYT?

Reply:
> Given that `MSYS2-packages` can only be built for `x86_64` and `i686` at this point (as opposed to `MINGW-packages`, which supports `aarch64` as well), shall I just restore the original logic for this arm?

Yes, please!

> And for the `mingw-w64-*` arm, I think I can just replace `$sdk64/usr/src/MINGW-packages/$name` with `/usr/src/MINGW-packages/$name`, right? Same for the `$sdk64/git-cmd.exe` invocations below. If that works, we also don't have to introduce the `sdkarm64` variable above. WDYT?

That sounds really good!